fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -452,7 +452,7 @@ public class Config extends ConfigBase {
 
     @ConfField(mutable = true, masterOnly = true, description = {"Insert load 的默认超时时间，单位是秒。",
             "Default timeout for insert load job, in seconds."})
-    public static int insert_load_default_timeout_second = 3600; // 1 hour
+    public static int insert_load_default_timeout_second = 14400; // 4 hour
 
```

Comment:
Documentation related has not been updated yet
https://doris.apache.org/docs/dev/data-operate/import/import-way/insert-into-manual/#fe-configuration